package.json

```diff
@@ -8,7 +8,9 @@
     "lint": "npm run lint --workspaces",
     "test": "npm run test --workspace=langium",
     "langium:generate": "npm run langium:generate --workspace=langium --workspace=domainmodel --workspace=arithmetics --workspace=statemachine",
-    "prepare": "npm run clean && npm run build"
+    "prepare": "npm run clean && npm run build",
+    "dev-build": "npm run dev-clean && npm install && npm link ./packages/langium && npm link ./packages/langium-cli && npm link ./packages/langium-vscode && npm link ./packages/generator-langium",
+    "dev-clean": "rimraf ./packages/**/node_modules && npm uninstall -g langium-workspaces langium-cli langium-vscode generator-langium langium && npm unlink langium-workspaces langium-cli langium-vscode generator-langium langium"
```

Comment:
`langium-vscode` will never be used locally as a package. We don't publish it via npm, but `vsce` / `ovsx`.

Reply:
Ok, then it can be removed here.

Reply:
And what is `langium-workspaces`?

Reply:
If you perform `npm link` in the root it links the whole langium workspace to the global npm folder. This is some sanity clean-up.